app/inpututils.cpp

```diff
@@ -270,6 +270,40 @@ bool InputUtils::cpDir( const QString &srcPath, const QString &dstPath )
   return true;
 }
 
+QString InputUtils::renameFile( const QString &srcPath, const QString &prefix )
+{
+  QFile file( srcPath );
+
+  if ( file.exists() )
```

Comment:
Just a nitpick, I find it easier to read with the static version `QFile::exists( srcPath )`  ... no need to look back what is 'file' :)